src/routines/get_results.jl

```diff
@@ -16,6 +16,24 @@ function _result_dataframe(variable::JuMP.Containers.DenseAxisArray)
 
 end
 
+function _result_dataframe_d(constraint::JuMP.Containers.DenseAxisArray)
+
+    result = Array{Float64, length(constraint.axes)}(undef, length(constraint.axes[1]))
+    # TODO: Remove this line once PowerSystems moves to Symbols
+    names = Array{Symbol, 1}(undef, length(constraint.axes[1]))
+
+    for (ix, name) in enumerate(constraint.axes[1])
+
+        result[ix] = JuMP.dual(constraint[name])/100
```

Comment:
We shouldn't do the hardcoded division here. The interpretation of the dual should depend on the user.

Reply:
Sure, I can remove that.